src/components/ActivityFeed/PostReactions.tsx

```
@@ -0,0 +1,198 @@
+					</div>
+				)}
+			</div>
+
+			<div className='flex items-center gap-1 md:gap-3'>
+				<div className='flex items-center gap-2'>
+					<span>
+						<Popover
+							placement='bottom'
+							trigger='hover'
+							content={<>{renderUsernames('👎')}</>}
+							arrow={true}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)